src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.cs

```diff
@@ -32,6 +30,22 @@ internal partial class NavigateToItemProvider : INavigateToItemProvider
             _displayFactory = new NavigateToItemDisplayFactory();
         }
 
+        public ISet<string> KindsProvided { get; } = ImmutableHashSet.Create(
+            NavigateToItemKind.Class,
+            NavigateToItemKind.Constant,
+            NavigateToItemKind.Delegate,
+            NavigateToItemKind.Enum,
+            NavigateToItemKind.EnumItem,
+            NavigateToItemKind.Event,
+            NavigateToItemKind.Field,
+            NavigateToItemKind.Interface,
+            NavigateToItemKind.Method,
+            NavigateToItemKind.Module,
+            NavigateToItemKind.Property,
+            NavigateToItemKind.Structure);
```

Comment:
there was an OtherSymbol right?  do we need that?

Reply:
We don't produce it 🤷‍♂️

Reply:
Ah ok.  So these are all the kinds we produce.  That works for me.